### PR TITLE
Close a bunch of canvas issues

### PR DESCRIPTION
**Pull request by eyuelt**
Fix the following issues:
- #65 Fonts don't fall back gracefully when Roboto is missing
- #66 Canvas doesn't resize with screen
- #67 Disable default mouse event behavior while dragging canvas
- #68 Support moving individual cards on canvas
- #70 Change canvas coordinate system's origin to be consistent with SVG
- #71 Get rid of SVG artifacts

### CODE REVIEW

File: src/client/app/shared/canvas/canvas.component.ts [hunk 1]
```diff
@@ -18,37 +18,53 @@ export class CanvasComponent {
...
 
   // TODO(eyuelt): should this use HostListener or the template event binding?
   onMousewheel(event: WheelEvent) {
-    const zoomScale = 1 + (event.deltaY * -0.01);
-    const zoomPnt = {x: event.offsetX, y: event.offsetY};
-    this.canvasService.zoom(zoomPnt, zoomScale);
     event.stopPropagation();
     event.preventDefault();
+    const zoomScale = 1 + (event.deltaY * -0.01);
+    const zoomPnt = {x: event.clientX - this.getBounds().left, y: event.clientY - this.getBounds().top};
```
Comment (girum11) [inline src/client/app/shared/canvas/canvas.component.ts]: Try to stick to 80 character line length, as per Google JS Style Guide?
https://google.github.io/styleguide/javascriptguide.xml

**Comment by girum11** [comment]
LGTM

File: src/client/app/shared/canvas/canvas.component.ts (hunk 1 above)
Comment (eyuelt) [inline src/client/app/shared/canvas/canvas.component.ts]: Done.

**Comment by eyuelt** [comment]
LGTM

**Pull request merged**